s64da_benchmark_toolkit/prepare.py

```diff
@@ -185,15 +185,15 @@ def _load_license(self, conn):
             print(f'S64 DA license status: {license_status}')
 
         except errors.UndefinedFunction as err:
-            print(f'License check function not found. Skipping, presumably on AWS.')
+            print('License check function not found. Skipping, presumably on AWS.')
             return
 
-        except errors.InternalError as err:
-            print(f'S64 DA licensing error: {err}')
-            raise
+        except errors.InternalError:
+            print(f'Could not load S64 DA license file or file is invalid: {license_path}')
+            return
```

Comment:
IMO at least the message is misleading. Especially since it will not stop loading data. Hence, the user has no guarantee that it will fully work.